blas/unit_test/Test_Blas1_team_mult.hpp

```diff
@@ -242,6 +243,8 @@ void impl_test_team_mult_mv(int N, int K) {
 
   typename Kokkos::ArithTraits<ScalarC>::mag_type const eps =
       Kokkos::ArithTraits<ScalarC>::epsilon();
+  typename Kokkos::ArithTraits<ScalarC>::mag_type const max_error =
```

Comment:
// The maximum rounding error for mult is dominated by `beta * x * y` for the sum in `gamma * y + beta * x * y`.